From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 21 Jun 2011 11:21:10 +0200
Subject: [PATCH] libcppunit needs -ldl dlopen/dlclose etc.

Without this building with -Wl,--as-needed fails later with
undefined references to dlopen/dlclose etc.

Bug: http://sourceforge.net/tracker/?func=detail&aid=3123759&group_id=11795&atid=111795
Forwarded: yes
Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/cppunit/Makefile.am |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/cppunit/Makefile.am b/src/cppunit/Makefile.am
index d5a60bf..449186e 100644
--- a/src/cppunit/Makefile.am
+++ b/src/cppunit/Makefile.am
@@ -65,3 +65,6 @@ libcppunit_la_LDFLAGS= \
  -no-undefined -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
  -release $(LT_RELEASE)
 
+libcppunit_la_LIBADD= \
+ $(LIBADD_DL)
+
